WindowsServerDocs/networking/windows-time-service/windows-server-2016-improvements.md

```diff
@@ -54,11 +54,11 @@ The following describes the changes in default configuration between Windows 201
 |**Standalone Client**||||
 | |*Time Server*|NA|time.windows.com|time.windows.com|
 | |*Polling Frequency*|NA|Once a day|Once a week|
-| |*Clock Update Frequency*|NA|Once a day|Once a week|
+| |*Clock Update Frequency*|NA|Once a day|Once a hour|

```

Comment:
I have not checked the validity of these values, but the grammar needs adaption:
```suggestion
| |*Clock Update Frequency*|NA|Once a day|Once an hour|
```
("an" is used before words starting with a vowel _when pronounced_ in English. The H in hour is silent, so it is pronounced like "our".)